core/static/static.go

```diff
@@ -72,7 +72,8 @@ func SetConsumerName(uri *url.URL, name string, id *uuid.UUID) {
 
 //nolint
 const (
-	EvmMaxInFlightTransactionsWarningLabel = `WARNING: If this happens a lot, you may need to increase ETH_MAX_IN_FLIGHT_TRANSACTIONS to boost your node's transaction throughput, however you do this at your own risk. You MUST first ensure your ethereum node is configured not to ever evict local transactions that exceed this number otherwise the node can get permanently stuck`
-	EvmMaxQueuedTransactionsLabel          = `WARNING: Hitting ETH_MAX_QUEUED_TRANSACTIONS is a sanity limit and should never happen under normal operation. This error is very unlikely to be a problem with Chainlink, and instead more likely to be caused by a problem with your eth node's connectivity. Check your eth node: it may not be broadcasting transactions to the network, or it might be overloaded and evicting Chainlink's transactions from its mempool. Increasing ETH_MAX_QUEUED_TRANSACTIONS is almost certainly not the correct action to take here unless you ABSOLUTELY know what you are doing, and will probably make things worse`
-	EthNodeConnectivityProblemLabel        = `WARNING: If this happens a lot, it may be a sign that your eth node has a connectivity problem, and your transactions are not making it to any miners`
+	EvmMaxInFlightTransactionsWarningLabel   = `WARNING: If this happens a lot, you may need to increase ETH_MAX_IN_FLIGHT_TRANSACTIONS to boost your node's transaction throughput, however you do this at your own risk. You MUST first ensure your ethereum node is configured not to ever evict local transactions that exceed this number otherwise the node can get permanently stuck. See the documentation for more details: https://docs.chain.link/docs/configuration-variables/`
```

Comment:
WDYT about grouping these in a package for just evm labels? `chains/evm/label`
Names can be shorter w/o qualifiers, and callers can do just: `label.MaxInFlightTransactionsWarning`
```suggestion
	MaxInFlightTransactionsWarning   = `WARNING: If this happens a lot, you may need to increase ETH_MAX_IN_FLIGHT_TRANSACTIONS to boost your node's transaction throughput, however you do this at your own risk. You MUST first ensure your ethereum node is configured not to ever evict local transactions that exceed this number otherwise the node can get permanently stuck. See the documentation for more details: https://docs.chain.link/docs/configuration-variables/`
```